### ISSUE TITLE
About the data preprocessing

### ISSUE THREAD

**Issue by derekshiii** [CSCYQJ/MICCAI23-ProtoContra-SFDA#12]
The work of the paper is very impressive
I would like to ask about the preprocessing of your dataset. I have seen that your dataset is not the original BTCV and CHAOS dataset in other issues, so I would like to ask what is the preprocessing of these data

**Comment by BarY7** [comment]
Hi @derekshiii , did you manage to recreate the preprocessing?
It seems the preprocessing used is similar to SIFA:

https://github.com/cchen-cc/SIFA?tab=readme-ov-file

But both in SIFA and in my implementation I don't seem to reach the same results as the paper reports

**Comment by Milk-White** [comment]
Hello@BarY7 , I downloaded the dataset used in SIFA and found that only the test set ends in. npz, while the training set is in other formats. May I know how your training set was processed, or do you have a link to a processed MR training set? Thank you!

**Comment by BarY7** [comment]
@Milk-White You can try looking in the preprocess notebook in the commit history. I pretty much gave up because even with the notebook its hard to reproduce the author's results, and they are not available here sadly

**Comment by Milk-White** [comment]
@BarY7 Thank you for your answer, thank you
